### PR TITLE
SAK-44610 Forums will not accept rubric scores with 3+ decimals

### PR DESCRIPTION
**Pull request by josecebe**

### CODE REVIEW

File: msgcntr/messageforums-app/src/webapp/js/forum.js [hunk 1]
```diff
@@ -877,6 +877,8 @@ $(document).ready(function(){
 
       const rubricGrading = document.getElementsByTagName("sakai-rubric-grading").item(0);
       rubricGrading && rubricGrading.save();
+      $(rubricGrading).find('input').each(function() {
```
Comment (adrianfish) [inline msgcntr/messageforums-app/src/webapp/js/forum.js:880]: rubricGrading && rubricGrading.querySelectorAll("input").forEach(i => i.prop("disabled", true));

Comment (adrianfish) [reply msgcntr/messageforums-app/src/webapp/js/forum.js:880]: What are you disabling here? Is it the score override boxes?

**Review by adrianfish — CHANGES_REQUESTED** [review]
You need to add the SAK number to the commit.

File: msgcntr/messageforums-app/src/webapp/js/forum.js (hunk 1 above)
Comment (adrianfish) [reply msgcntr/messageforums-app/src/webapp/js/forum.js:880]: If it is the comment boxes, you should probably be more specific
"input.fine-tune-points"

**Comment by adrianfish** [comment]
The rubric is accepting more than 2 decimal places everywhere, and this is wrong. Try it in assignments. I think this PR should be closed and the JIRA should be rewritten to reflect this. This is a regression, basically.

**Comment by josecebe** [comment]
> The rubric is accepting more than 2 decimal places everywhere, and this is wrong. Try it in assignments. I think this PR should be closed and the JIRA should be rewritten to reflect this. This is a regression, basically.

Hi @adrianfish , in forums it is not working as expected (you can check it in the video attached in JIRA ticket). The rubric webcomponent is inside a form on this tool, so disabling these inputs it's just a trick to avoid these inputs to be validated using the forums form originally criteria.

**Comment by adrianfish** [comment]
@josecebe But if you actually fix the rubric and stop it accepting more than 2dp, then it will work everywhere. That way you are fixing the underlying issue, not the symptom.

**Comment by josecebe** [comment]
@adrianfish I see what you say, I thought it was the correct behaviour to allow more than 2 decimals. I'm going to work on a proper solution.

**Comment by adrianfish** [comment]
@josecebe I'm pretty sure that was the behaviour, just allowing 2 d.p. I think something may have broken that. Take a look at the rating code in the actual rubrics manager parts. I don't think that allows more than 2 dp, so hopefully that's still working. Thanks for looking at it.

**Comment by ern** [comment]
@josecebe can you resolve the conflicts

**Comment by mpellicer** [comment]
Closing and unassigning @josecebe

**Pull request closed without merging**